web/src/LoginPage/LoginPage.jsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { connect } from 'react-redux';
+
+import { userActions } from '../_actions';
+import '../_designs/design.css'
+import '../_designs/login.css'
+
+class LoginPage extends React.Component {
+    constructor(props) {
+        super(props);
+
+        // reset login status
+        this.props.dispatch(userActions.logout());
+
+        this.state = {
+            username: '',
+            password: '',
+            submitted: false
+        };
+
+        this.handleChange = this.handleChange.bind(this);
+        this.handleSubmit = this.handleSubmit.bind(this);
+    }
+
+    handleChange(e) {
```

Comment:
in case you want to use arrow functions since they execute in their parent context, you can remove the `.bind(this)`

```suggestion
    handleChange = (e) => {
```